test/parallel/test-fs-constants.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+require('../common');
+const fs = require('fs');
+const assert = require('assert');
+
+// Check if the two constants accepted by chmod() on Windows are defined.
+assert.notStrictEqual(fs.constants.S_IRUSR, undefined);
```

Comment:
Nit: If this is Windows specific, would it be better to define the expected behaviour based on the OS in which the test is run? Perhaps `require('../common').isWindows` could be used here?

Reply:
I think this is supposed to test that `S_IRUSR` and `S_IWUSR` are available both on Windows (exposed in this PR) as well as other platforms (which were already there from before), so the current test is okay IMO.